test/bigtest/network/config.js

```diff
@@ -190,7 +190,7 @@ export default function config() {
     }]
   }]);
   // tags endpoint
-  this.get('/tags', {
+  this.get('/tags', ({
```

Comment:
I would prefer to configure models, factories etc so that mirage can support tags. This way you won't have to hardcode tag names in tests. Also tags functionality would become available in mirage mode, now it's not